### PR TITLE
:art: Improve lodash imports with babel-plugin-lodash

### PR DESCRIPTION
**Pull request by kefranabg**
See #161

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
 module.exports = {
-  presets: [['@vue/app', { useBuiltIns: 'entry' }]]
+  plugins: ['lodash'],
+  presets: [
```
Comment (tbetous) [inline babel.config.js]: Why did you add @babel/env preset ?

Comment (kefranabg) [reply babel.config.js]: You're right, I can remove it
